### ISSUE TITLE
VS range doesn't have `make_iterator_range`

### ISSUE THREAD

**Issue by tower120** [Microsoft/Range-V3-VS2015#14]
VS version of range-v3 doesn't have `make_iterator_range`, instead it uses `make_range` (which standard ranges-v3 do not have).

Due to this you can't have crosscompiler code with ranges...

**Comment by CaseyCarter** [comment]
> Due to this you can't have crosscompiler code with ranges...

I agree the portability impact of this name difference is large. I'll take a look at specifically updating this part of the library from upstream.
